src/lib/eDSL.ml

```diff
@@ -302,18 +301,25 @@ let depends_on l =  Depends_on l
 let on_success_activate n = On_success_activate n
 let on_failure_activate n = On_failure_activate n
 
+type done_when_option = [
+  | `Dependencies_are
+  | `Is_verified of Condition.t
+  | `Product_is 
```

Comment:
I added a little blob to force the eyes in the rights direction:
<img width="457" alt="screenshot 2016-04-28 13 24 03" src="https://cloud.githubusercontent.com/assets/617111/14895008/8b529382-0d44-11e6-8870-7dfaba979976.png">